app/GraphQL/Guild/Mutations/Leads/SocialMutation.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\GraphQL\Guild\Mutations\Leads;
+
+use Kanvas\Guild\Leads\Actions\RemoveLeadParticipantAction;
+use Kanvas\Guild\Leads\DataTransferObject\LeadsParticipant;
+use Kanvas\Guild\Leads\Models\Lead;
+use Kanvas\Social\Follows\Models\UsersFollows;
+
+class SocialMutation
+{
+    /**
+     * follow a lead
+     */
+    public function follow(mixed $root, array $req): bool
+    {
+        $lead = Lead::getByUuidFromBranch(
```

Comment:
this method, is there in a trait o base model ?
Is a nice idea, more model can use this in the future

Reply:
This is in the base model